source/assets/scripts/backend.js

```diff
@@ -323,6 +382,7 @@ export async function search_recipe(name, online=false, match_tolerance=10, into
   for (let i = 0; i < localStorage.length; i++) {
     const recipe_hash = localStorage.key(i);
     if (recipe_hash.startsWith('%')) continue;  // ignore
+    if (recipe_hash.startsWith('!')) continue;
```

Comment:
the image will be stored with the key start by !, we don't want to look through it.